sse2neon.h

```diff
@@ -2359,6 +2362,24 @@ FORCE_INLINE __m128 _mm_add_ps(__m128 a, __m128 b)
         vaddq_f32(vreinterpretq_f32_m128(a), vreinterpretq_f32_m128(b)));
 }
 
+// Add packed double-precision (64-bit) floating-point elements in a and b, and
+// store the results in dst.
+// https://software.intel.com/sites/landingpage/IntrinsicsGuide/#text=_mm_add_pd
+FORCE_INLINE __m128d _mm_add_pd(__m128d a, __m128d b)
+{
+#if defined(__aarch64__)
+    return vreinterpretq_m128d_f64(
+        vaddq_f64(vreinterpretq_f64_m128d(a), vreinterpretq_f64_m128d(b)));
+#else
+    double *aptr = (double *) &a;
```

Comment:
Simply use `da` and `db` rather than annoying `aptr` and `bptr`.

Reply:
Agree.